Card/src/main/java/com/braintreepayments/api/CardClient.java

```diff
@@ -95,17 +96,22 @@ private void handleTokenizeResponse(JSONObject tokenizationResponse, Exception e
         if (tokenizationResponse != null) {
             try {
                 CardNonce cardNonce = CardNonce.fromJSON(tokenizationResponse);
-
-                callback.onCardResult(new CardResult.Success(cardNonce));
-                braintreeClient.sendAnalyticsEvent("card.nonce-received");
-
+                callbackSuccess(callback, new CardResult.Success(cardNonce));
             } catch (JSONException e) {
-                callback.onCardResult(new CardResult.Failure(e));
-                braintreeClient.sendAnalyticsEvent("card.nonce-failed");
+                callbackFailure(callback, new CardResult.Failure(e));
             }
         } else if (exception != null) {
-            callback.onCardResult(new CardResult.Failure(exception));
-            braintreeClient.sendAnalyticsEvent("card.nonce-failed");
```

Comment:
Were these existing events already being sent to FPTI? Just want to make sure that this change doesn't break any existing reports.

Reply:
As far as I know, there aren't any existing reports that are based on the old events, and the new dashboards will be based on these new planned events - but @scannillo correct me if I'm wrong.

Reply:
That's a great question. So on this V5 branch, we already [merged in a change](https://github.com/braintree/braintree_android/pull/834) that sends our analytics directly to FPTI versus sending to the old BT analytics service (Arachne).

The old events weren't being used in any dashboard, but these new events now mirror the ones on BT iOS V6, and will be able to be used in our new conversion dashboard that we share b/w BT & PPCP mobile.

With Android V5, we're doing this analytics cleanup to keep things consistent across platform & product